terraform/deploy/modules.tf

```diff
@@ -32,8 +32,12 @@ data "aws_secretsmanager_secret_version" "dataworks" {
 }
 
 locals {
-  amazon_region_domain  = "${data.aws_region.current.name}.amazonaws.com"
-  endpoint_services     = ["secretsmanager", "ec2messages", "s3", "monitoring", "ssm", "ssmmessages", "ec2", "kms", "logs", "api.ecr", "dkr.ecr", "ecs", "elasticloadbalancing", "events", "application-autoscaling", "kinesis-firehose"]
+  amazon_region_domain = "${data.aws_region.current.name}.amazonaws.com"
+  endpoint_services = join("", concat(
```

Comment:
`=` aren't aligned :(

Reply:
huh, not sure how it passed two tf fmt checks and wasn't aligned

Reply:
for future viewers, strangely, terraform format actually put it like this, and correcting it to @mattburgess 's very correct suggestion (which is what I'd originally done it as), fails the tf format check.